Lib/sqlite3/dbapi2.py

```diff
@@ -85,3 +87,11 @@ def convert_timestamp(val):
 # Clean up namespace
 
 del(register_adapters_and_converters)
+
+deprecated_names = ["version", "version_info"]
```

Comment:
Since this module doesn't define `__all__`, it would be better to have this be called `_deprecated_names` (starting with a single underscore), as otherwise it's an implicitly public name (but it shouldn't be — end users shouldn't be accessing it, as it's an implementation detail)

Reply:
if I make `deprecated_names` private(`_deprecated_names`), it won't be importable into `sqlite3.__init__` I guess it is ok to re declare `_deprecated_names` again in __init__.py?

Reply:
I think i got it. I  can just import them into `__init__.py`